layouts/terms/single.html

```diff
@@ -15,9 +15,16 @@ <h1>{{ .Title }}</h1>
 
       {{ .Content }}
 
+      {{ if isset .Params "synonyms" }}
+        <h2>Synonyms</h2>
+        <li>
+        <ul>{{ delimit .Params.synonyms ", " ", and " }}</ul>
```

Comment:
You've got `<ul>` and `<li>` reversed here.

Reply:
Oh crap. Brainfart.

Reply:
Fixed it.